internal/cmd/destinations.go

```diff
@@ -82,16 +87,24 @@ func newDestinationsRemoveCmd() *cobra.Command {
 				return err
 			}
 
+			logging.S.Debug("call server: list destinations named [%s]", args[0])
 			destinations, err := client.ListDestinations(api.ListDestinationsRequest{Name: args[0]})
 			if err != nil {
 				return err
 			}
 
 			if destinations.Count == 0 {
-				return fmt.Errorf("no destinations named %s.", args[0])
+				if force {
+					return nil
+				}
+				return Error{
```

Comment:
is this a user-facing error? I think perhaps we should change the name of the type to something more descriptive? at the moment it seems like `Error` is appropriate to use for all errors, but I'm assuming it's only appropriate for errors where you want to customize the text to be something the user sees and acts on. This probably differs from other errors in that these shouldn't be wrapped, prefixed, or displayed with stack traces, etc. thoughts?

Reply:
We initially named it a "userfacing error" or some sort, but then after some discussion, a few of us were okay with naming it "error", since if we named it "UserFacingError" we don't need "Error" right now anyways (in the future we could rename if it becomes more robust and we need to split them)